server/portal/apps/system_monitor/views.py

```diff
@@ -47,61 +44,30 @@ class System:
 
     def __init__(self, system_dict):
         try:
+            self.display_name = system_dict.get('display_name')
             self.hostname = system_dict.get('hostname')
-            self.display_name = system_dict.get('displayName')
-            if 'ssh' in system_dict.keys():
-                self.ssh = system_dict.get('ssh')
-            if 'heartbeat' in system_dict.keys():
-                self.heartbeat = system_dict.get('heartbeat')
-            if 'tests' in system_dict.keys():
-                self.status_tests = system_dict.get('tests')
-            if 'jobs' in system_dict.keys():
-                self.resource_type = 'compute'
-                self.jobs = system_dict.get('jobs')
-                self.load_percentage = system_dict.get('load')
-                if isinstance(self.load_percentage, (float, int)):
-                    self.load_percentage = int((self.load_percentage * 100))
-                else:
-                    self.load_percentage = None
-                self.cpu_count = system_dict.get('totalCpu')
-                self.cpu_used = system_dict.get('usedCpu')
+            self.resource_type = 'compute'
+            self.running = system_dict.get('running')
+            self.waiting = system_dict.get('waiting')
+            self.load = system_dict.get('load')
+            if isinstance(self.load, (float, int)):
+                self.load = int((self.load * 100))
             else:
-                self.resource_type = 'storage'
-                self.cpu_count = 0
+                self.load = 0
+            self.online = system_dict.get('online')
+            self.reachable = system_dict.get('reachable')
+            self.queues_down = system_dict.get('queues_down')
+            self.in_maintenance = system_dict.get('in_maintenance')
             self.is_operational = self.is_up()
         except Exception as exc:
             logger.error(exc)
 
     def is_up(self):
-        '''
-        Checks each uptime metric to determine if the system is available
-        '''
         if self.resource_type == 'compute':
```

Comment:
We can get rid of this check, since we assume we're dealing with `compute` systems

Reply:
Could also simplify the if/else:
```
def is_up(self):
        return self.online and self.reachable and not (self.queues_down or self.in_maintenance)
```